test/playwright/Stock/AxisResizer/axis-resizer.spec.ts

```diff
@@ -0,0 +1,33 @@
+// create a test in playwright that checks if the axis resizer is working
+import { test, expect } from '@playwright/test';
+declare const Highcharts: any;
+
+test('axis resizer gui is working', async ({ page }) => {
+    const urlPrefix = 'view?path=';
+    await page.goto(`${urlPrefix}highcharts/cypress/stock-tools-gui`);
```

Comment:
I'm curious if we can make kind of utility function, which could get as an argument a short path to test and return the full path, like this: `page.goto(pathToTest('stock-tools-gui'))`. 

PS: `cypress` in path? 🤔

Reply:
Will took into that!
The `cypress` is there, because I decided to use the samples that are there for cypress testing, and I think that they can stay there. We can rename the directory later.